### PR TITLE
config_tools: rename UOS to user vm in UI

### PR DESCRIPTION
**Pull request by Kunhui-Li**
Since PRs #6760 and #6765 have landed,
we also need to rename "UOS" to "user vm" in UI.

Tracked-On: #6744
Signed-off-by: Kunhui-Li <kunhuix.li@intel.com>

### CODE REVIEW

File: misc/config_tools/config_app/templates/launch.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h4 class="modal-title" id="myModalLabel">Save Launch XML</h4>
         <div class="modal-content">
             <div class="modal-header">
                 <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
-                <h4 class="modal-title" id="addlaunchModalLabel">Configure UOS for Post-launched VM</h4>
+                <h4 class="modal-title" id="addlaunchModalLabel">Configure user vm for Post-launched VM</h4>
```
Comment (gvancuts) [inline misc/config_tools/config_app/templates/launch.html:46]:
```suggestion
                <h4 class="modal-title" id="addlaunchModalLabel">Configure User VM for Post-launched VM</h4>
```

File: misc/config_tools/config_app/templates/launch.html [hunk 2]
```diff
@@ -146,7 +146,7 @@ <h4 class="modal-title" id="addlaunchModalLabel">Configure UOS for Post-launched
                 </div>
                 <div class="form-group">
                     <button type="button" class="col-sm-6 btn" id="add_launch_vm" data-id="{{vm.attrib['id']}}" data-toggle="modal"
-                            data-target="#add_launch_modal">Configure an UOS below</button>
+                            data-target="#add_launch_modal">Configure a user vm below</button>
```
Comment (gvancuts) [inline misc/config_tools/config_app/templates/launch.html:149]:
```suggestion
                            data-target="#add_launch_modal">Configure a User VM below</button>
```

File: misc/config_tools/config_app/templates/launch.html [hunk 3]
```diff
@@ -394,7 +394,7 @@ <h4 class="modal-title" id="addlaunchModalLabel">Configure UOS for Post-launched
         {% endfor %}
         <tr><td>
             <button type="button" class="btn" id="add_launch_script" data-toggle="modal" data-id="-1"
-                            data-target="#add_launch_modal">Configure an UOS</button>
+                            data-target="#add_launch_modal">Configure a user vm</button>
```
Comment (gvancuts) [inline misc/config_tools/config_app/templates/launch.html:397]:
```suggestion
                            data-target="#add_launch_modal">Configure a User VM</button>
```

**Review by gvancuts — CHANGES_REQUESTED** [review]

File: misc/config_tools/config_app/templates/launch.html (hunk 3 above)
Comment (Kunhui-Li) [reply misc/config_tools/config_app/templates/launch.html]: Updated, thanks

File: misc/config_tools/config_app/templates/launch.html (hunk 2 above)
Comment (Kunhui-Li) [reply misc/config_tools/config_app/templates/launch.html]: Updated, thanks.

**Review by gvancuts — APPROVED** [review]

**Review by NanlinXie — APPROVED** [review]

**Pull request merged**